cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/stats/BrokerUtilizationStats.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2018 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
```

Comment:
2019

Reply:
This class is technically created in 2018 (the old BrokerStats class), do we still need an update?

Reply:
Update just to reflect how it was updated as part of our change.